tests/Entity/CurriculumInventoryAcademicLevelTest.php

```diff
@@ -84,26 +84,27 @@ public function testSetReport()
     }
 
     /**
-     * @covers \App\Entity\CurriculumInventoryAcademicLevel::addSequenceBlock
+     * @covers \App\Entity\CurriculumInventoryAcademicLevel::addStartingSequenceBlock
```

Comment:
Should probably be duplicated for `endingSequenceBlock` just to be sure (same for remove test below)

Reply:
good catch. i added those getter/setter tests :heavy_check_mark: